packages/theming/demo/utilities.stories.mdx

```diff
@@ -78,7 +79,8 @@ import README from '../README.md';
       isAnimated: true
     }}
     argTypes={{
-      position: { control: { type: 'radio', options: MENU_POSITIONS } }
+      position: { control: 'radio', options: MENU_POSITIONS },
```

Comment:
These `select` and `radio` controls were not working due to a previous Storybook update.